plugins/python/python_plugin.c

```diff
@@ -242,6 +242,10 @@ int uwsgi_python_init() {
 	}
 
 	if (up.home != NULL) {
+		if (!uwsgi_is_dir(up.home)) {
+			uwsgi_log("Python Home is not a directory: %s\n", up.home);
+			exit(1);
```

Comment:
That was more of a question than a suggestion but i see we're racing each other :) I'm thinking of people that does not test their config before putting into production that may have the site working outside the home.

Reply:
Are you suggesting it's possible this works in some situations even with an invalid value?

Reply:
Am happy to make this a lower impact patch that simply logs, and doesn't exit.

Reply:
may i suggest to have it only print an error in the logs for the 2.0 branch and to brutally exit (like the one you proposed) in master ? (where we suppose to move our choices to 2017 best-practices  instead of 2009 ;)

Reply:
Sounds perfect to me :)
Shall I leave it all to you from here, then? The difference being a single line.

Reply:
yep, i'll do it